.github/CODEOWNERS

```diff
@@ -60,6 +60,6 @@
 /plugins/dominant-color-images              @pbearne @spacedmonkey
 /tests/plugins/dominant-color-images        @pbearne @spacedmonkey
 
-# Plugin: Image Loading Optimization
-/plugins/image-loading-optimization         @westonruter
-/tests/plugins/image-loading-optimization   @westonruter
+# Plugin: Optimization Detective
+/plugins/optimization-detective             @westonruter
+/tests/plugins/optimization-detective       @westonruter
```

Comment:
Feel free to add me as second code owner here. While I've mostly been just supporting/reviewing, I'd like to remain involved on this one.

Reply:
```suggestion
/plugins/optimization-detective             @westonruter @felixarntz
/tests/plugins/optimization-detective       @westonruter @felixarntz
```